packages/teams-js/src/public/geoLocation.ts

```diff
@@ -64,7 +64,7 @@ export namespace geoLocation {
     }
     const permissions: DevicePermission = DevicePermission.GeoLocation;
 
-    return new Promise<boolean>(resolve => {
+    return new Promise<boolean>((resolve) => {
```

Comment:
These prettier fixes annoy me. You did not touch this file and it still make changes here. I hope somebody else's setting don't put this parenthesis back in next PR

Reply:
I can understand this trying to fix it